commands/analyze.go

```diff
@@ -39,109 +39,142 @@ func init() {
 
 func analyze(inDb string, configFile string) error {
 	res := resources.InitResources(configFile)
-	var toRunDirty []string
-	var toRun []string
+	var toRunDirty []database.RITADatabase
+	var toRun []database.RITADatabase
 
 	// Check to see if we want to run a full database or just one off the command line
 	if inDb == "" {
-		res.Log.Info("Running analysis against all databases")
-		toRunDirty = append(toRun, res.MetaDB.GetUnAnalyzedDatabases()...)
```

Comment:
A possible error is hidden in the old code as res.MetaDB.GetUnAnalyzedDatabases may fail